skills/studio/scripts/studio/utils/doc_index.py

```
@@ -0,0 +1,378 @@
+
+    old_sections = cached["retrieval_sections"]
+    if len(old_sections) != len(fresh_sections):
+        return {
+            "structural_change": True,
+            "unchanged": [],
+            "changed": [s["heading"] for s in fresh_sections],
+        }
+
+    unchanged: List[str] = []
+    changed: List[str] = []
+    for old, new in zip(old_sections, fresh_sections):
```

Ruff (0.16.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)